setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='edx-opaque-keys',
-    version='0.3.1',
+    version='0.3.2',
```

Comment:
I don't see a changelog for this package; shouldn't it have one?  (I like to add a comment on this line reminding developers to update it when bumping the version.)